apps/design/backend/migrations/1742248030415_add-ballot-language-codes.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * @type {import('node-pg-migrate').ColumnDefinitions | undefined}
+ */
+exports.shorthands = undefined;
+
+/**
+ * @param pgm {import('node-pg-migrate').MigrationBuilder}
+ * @param run {() => void | undefined}
+ * @returns {Promise<void> | void}
+ */
+exports.up = (pgm) => {
+  pgm.addColumn(
+    'elections',
+    {
+      ballot_language_codes: {
```

Comment:
I noticed that this column is already in the initial migration. Why have a separate migration for it as well?